src/System.Net.HttpListener/tests/HttpRequestStreamTests.cs

```diff
@@ -282,6 +282,34 @@ public void Dispose()
             }
         }
 
+        [Theory]
+        [InlineData(true)]
+        [InlineData(false)]
+        public async Task Read_Disposed_ReturnsZero(bool transferEncodingChunked)
+        {
```

Comment:
How about enabling the tests that differ in behavior for HttpRequestStream, this unification should have fixed some of them no?